app/agents/tools.py

```diff
@@ -15,8 +15,7 @@ def make_tools(llm: BaseLanguageModel, verbose=True):
         SMILES2Weight(),
         FuncGroups(),
         ExplosiveCheck(),
-        ControlChemCheck(),
-        SemanticSearch()#,
```

Comment:
@combspk , you are removing changes from my last PR. Could you keep this  line in? Thanks!

Reply:
Sorry, should be resolved now, forgot to pull first